goboy/emu.go

```diff
@@ -508,3 +508,12 @@ func (g *GameBoy) interruptJumpHelper(target uint16) {
 	g.mainMemory.write(g.get16Reg(SP), lowVal)
 	g.regs[PC] = target
 }
+
+func (Gb *GameBoy) GPULoop() {
+	for _ = range Gb.gpuClock.C {
+		// NOTE: Check debugger pause flag here
+		LY := Gb.mainMemory.read(0xff44)
```

Comment:
Since we know exactly where we're reading from and writing to, we may as well bypass the .read() and .write()